core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -853,4 +853,44 @@ class WorkspaceServiceSpec extends FlatSpec with ScalatestRouteTest with Matcher
     }
 
   }
+
```

Comment:
I wasn't sure where to do the tests; should I have some in workspaceApiSpec and the AttributeComponent for the query? Or is that redundant?